src/main/java/iana/tasks/Task.java

```diff
@@ -0,0 +1,109 @@
+package iana.tasks;
+
+import iana.exception.IanaException;
+
+public class Task {
```

Comment:
Hmm the making sense of the user control could be handled by the Parser class instead